src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Common/TensorPrimitives.IAggregationOperator.cs

```diff
@@ -1227,9 +1227,12 @@ static T Vectorized128(ref T xRef, ref T yRef, nuint remainder)
 
                         // We need to the ensure the underlying data can be aligned and only align
                         // it if it can. It is possible we have an unaligned ref, in which case we
-                        // can never achieve the required SIMD alignment.
+                        // can never achieve the required SIMD alignment. This cannot be done for
```

Comment:
There are other Aggregate overloads that do alignment. Those aren't affected?

Reply:
No those need fixing as well, I just didn't scroll enough in the file. Will update.

-- The general issue is that Aggregate operations are doing "across" and that means for floating-point they can be impacted by alignment because for one input we may do `(a + b) + (c + d)` and for another we may do `(0 + a) + (b + c) + (d + 0)`.

In general we aren't overly worried about minor differences for different inputs or across different hardware; that at least can be predicted and well-handled. But for two identical inputs to produce different results just because the GC relocated it implicitly to a new location, that's a bit more problematic 😄